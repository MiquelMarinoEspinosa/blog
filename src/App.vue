<template>
  <div>
    <app-header></app-header>
    <router-view></router-view>
  </div>
</template>

<script>
import Header from "./components/Header.vue";

export default {
  components: {
    "app-header": Header,
  },
  data() {
    return {};
  },
  methods: {},
};
</script>

<style>
body {
  margin: 0;
  font-family: "Nunito Semibold";
}
</style>
