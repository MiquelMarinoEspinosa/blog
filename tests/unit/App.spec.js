import { shallowMount, createLocalVue } from "@vue/test-utils";
import Router from "vue-router";
import App from "@/App";
import Header from "@/components/Header";

const localVue = createLocalVue();
localVue.use(Router);

describe("App", () => {
  it("renders the App", () => {
    const app = shallowMount(App, { localVue });
    expect(app.find(Header).exists()).toBeTruthy();
    expect(app.html()).toMatchSnapshot();
  });
});
