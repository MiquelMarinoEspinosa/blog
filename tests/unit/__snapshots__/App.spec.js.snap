// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the App 1`] = `
<div>
  <app-header-stub></app-header-stub>
  <router-view-stub name="default"></router-view-stub>
</div>
`;
